library/exceptions.po

```diff
@@ -439,8 +439,8 @@ msgstr ""
 "La deuxième forme du constructeur définit les attributs correspondants, "
 "décrits ci-dessous. Les attributs par défaut sont :const:`None` si non "
 "spécifiés. Pour la rétrocompatibilité, si trois arguments sont passés, "
-"l'attribut :attr:`~BaseException.args` contient seulement un *tuple* à deux "
-"valeurs des deux premiers arguments du constructeur."
+"l'attribut :attr:`~BaseException.args` contient seulement un *n*-uplet à "
```

Comment:
Un couple ?

Reply:
Ce qui me dérange un peu, c'est qu'on « perd » le fait de dire explicitement qu'il s'agit d'un _n_-uplet (et pas une séquence d'un autre type).
La source dit bien _a 2-tuple of the first two constructor arguments_, et non _the first two constructor arguments_

Reply:
Ça ne se jsutifie pas plus que dans les autres cas en fait. Je mets « paire » aussi ici.